dashboard/client/src/common/JobStatus.tsx

```diff
@@ -39,15 +39,18 @@ const useJobRunningIconStyles = makeStyles((theme) =>
   }),
 );
 
-type JobRunningIconProps = { small?: boolean } & ClassNameProps;
+type JobRunningIconProps = { small?: boolean } & ClassNameProps &
+  DataTestIdProps;
 
 export const JobRunningIcon = ({
   className,
   small = false,
+  "data-testid": dataTestId,
```

Comment:
nit: if possible, if there's an `...IconProps` type in react-icons, i'd prefer that we use `...iconProps` instead of `"data-testid"`.

Then it can be applied to the `<Ri...` component as
```typescript
  return (
    <RiLoader4Line
        ...iconProps
```
and the caller can still do
```typescript
<JobRunningIcon data-testid="blah"/>
```

Reply:
<img width="863" alt="Screenshot 2024-01-11 at 10 07 03 PM" src="https://github.com/ray-project/ray/assets/711935/7ead9f9a-1f1c-4bf5-b0c7-1dd5e223628b">

didn't seem to work, but I can try to make the test work with a11y attributes instead of data-testid.